src/Hungary/Mews.Fiscalizations.Hungary/Models/TaxPayer/TaxPayerData.cs

```diff
@@ -4,12 +4,13 @@ namespace Mews.Fiscalizations.Hungary.Models
 {
     public sealed class TaxPayerData
```

Comment:
Nope, the endpoint we use for this is to query a tax id and the government would return us some information about that tax id.

so it could be anyone's tax id.